website/www/site/assets/js/fix-playground-nested-scroll.js

```diff
@@ -10,10 +10,10 @@
 // License for the specific language governing permissions and limitations under
 // the License.
 
-$('#playgroundIframeContainer').on('click', function(_) {
-  $('#playgroundIframe').css('pointer-events', 'auto');
+$('.playground-iframe-wrapper-no-scroll').on('click', function(_) {
+  $('.playground-iframe-wrapper-no-scroll').addClass('playground-iframe-wrapper-scroll');
```

Comment:
This should be done on `$(this)`, otherwise we affect all Playground instances on the page.

Reply:
changed